db/schema.rb

```diff
@@ -77,6 +77,7 @@
     t.string "state"
     t.string "employer"
     t.string "position"
+    t.string "availability"
```

Comment:
I am not sure why this is here, and not the checkbox columns.  I suggest that you do a db:reset followed by a db:migrate to create the schema.rb as it should be.